docs/general/ddl/fulltext-indices.rst

```diff
@@ -42,8 +42,6 @@ Disable indexing
 ================
 
 Indexing can be turned off by using the ``INDEX OFF`` column definition.
-Consider that a column without an index can only be used as a result column
```

Comment:
I think it would be worth to add a note that disabling the index will result in poorer performance.